lib/client/api.go

```diff
@@ -4085,7 +4085,7 @@ func (tc *TeleportClient) Ping(ctx context.Context) (*webclient.PingResponse, er
 
 	// Verify server->client and client->server compatibility.
 	if tc.CheckVersions {
-		if !utils.MeetsVersion(teleport.Version, pr.MinClientVersion) {
+		if !utils.MeetsMinVersion(teleport.Version, pr.MinClientVersion) {
```

Comment:
Hmm, it's because `webclient.Ping` above returns an empty `pr.MinClientVersion` string (so `utils.MeetsVersion` returns `true`). Do you maybe know why it happens?